unlock-app/src/components/interface/buttons/Button.js

```diff
@@ -29,6 +29,58 @@ export const LayoutButton = ({ href, title, children, action, ...props }) => {
   return button
 }
 
+export const MobileToggleButton = ({
```

Comment:
Probably worth defining in its own file using the same approach as the other buttons.